holoviews/tests/element/test_raster.py

```diff
@@ -48,6 +48,11 @@ def test_construct_from_dict_with_alpha(self):
         rgb = RGB({'x': [1, 2, 3], 'y': [1, 2, 3], ('R', 'G', 'B', 'A'): self.rgb_array})
         self.assertEqual(len(rgb.vdims), 4)
 
+    def test_not_using_class_variables_vdims(self):
```

Comment:
I tested it before I pushed and tested it again on branch v1.14.1, where it failed. 

![image](https://github.com/holoviz/holoviews/assets/19758978/e8ce7e85-56f3-4177-8631-af73fadefc4e)

I used `is` instead of `==` to verify the identity and not equality. If I had used `==` It would also have passed before.

Reply:
Oh yes poor me ... sorry for the noise!